### ISSUE TITLE
Hero component - macro issues

### ISSUE THREAD

**Issue by boxadesign** [ONSdigital/design-system#2153]
Writing jest tests for the hero component highlighted a number of issues with the macro:

- ~The parameter `wide` is not documented in macro options.~
- We should move away from using things like `censusTheme` and just use something like `branded: true` and let css theme settings handle the look and feel.
- Also...
```
{% if params.censusThemeDark is defined and params.censusThemeDark %}
    {% set skinName = "dark" %}
{% else %}
    {% set skinName = "light" %}
{% endif %}
```
The above adds a suffix string to the image `src` of a pre title image. We should update the variable keys to make this clear. Or implement as a ternary inline in the src (perhaps).

**Comment by jrbarnes9** [comment]
First point fixed in #2126

**Comment by yatesn** [comment]
Branding has been removed as part of wider piece of work
